pom.xml

```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta4/</fuse-tooling-url>
+    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```

Comment:
Yes.  This was the part of my original PR that needs to be a separate Jira.  I suggested waiting
   until Fuse Tooling goes .Final before changing it.

    --paull



----- Original Message -----
> From: "Lars Heinemann" <notifications@github.com>
> To: "jboss-switchyard/tools" <tools@noreply.github.com>
> Cc: "Paul Leacu" <pleacu@redhat.com>, "Mention" <mention@noreply.github.com>
> Sent: Tuesday, September 6, 2016 10:00:21 AM
> Subject: Re: [jboss-switchyard/tools] FUSETOOLS-2074 - update 2.1.x for latest bpmn2 modeler (#600)
> 
> > @@ -23,7 +23,7 @@
> >      <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
> >      
> >      <!-- Fuse version -->
> > -
> > <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta4/</fuse-tooling-url>
> > +
> > <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
> 
> We do a downgrade?
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/jboss-switchyard/tools/pull/600/files/0af89509b68fcf79b0c78eedc1aafc7fb426cf65..bb38f37ac4502c9d355929484638e6c2e4e8f6c2#r77639713